### PR TITLE
Fixed multiple call on language change.

### PR DESCRIPTION
**Pull request by NayanKhedkar**
Refer:[Issue](https://github.com/adaptlearning/adapt_framework/issues/2368)

### CODE REVIEW

**Comment by moloko** [comment]
I had a thought that maybe we could get PageLevelProgressNavigationView to do `this.remove` on `app:languageChange` - but it doesn't quite seem to be doing the job. I haven't exactly spent a lot of time on it (and now have to do something else) but still feel it might be a valid solution?

**Comment by moloko** [comment]
OK, I think changing the initialize function adapt-contrib-pageLevelProgress.js to this:
```js
Adapt.on({
    'app:dataReady': this.onDataReady.bind(this),
    'app:languageChanged': function() {
        this.stopListening();
    }.bind(this)
});
```
If, like me, you look at that code and are tempted to tidy it up, be aware that if you do either this:
```js
Adapt.on({
    'app:dataReady': this.onDataReady.bind(this),
    'app:languageChanged': this.stopListening.bind(this)
});
```
or this:
```js
Adapt.on({
    'app:dataReady': this.onDataReady,
    'app:languageChanged': this.stopListening
}, this);
```
then both those event handlers get removed by `this.stopListening` as well...! my brain's too broken now to figure out why.

**Comment by NayanKhedkar** [comment]
@moloko  I am also wondering.
`` 'app:languageChanged': this.stopListening.bind(this) ``

`` 'app:languageChanged': function() {
        this.stopListening();
    }.bind(this) ``

both these two line doing same thing.May be the mess with scope.

``` 
should I go with.
Adapt.on({
    'app:dataReady': this.onDataReady.bind(this),
    'app:languageChanged': function() {
        this.stopListening();
    }.bind(this)
}) ;
```

**Comment by moloko** [comment]
> both these two line doing same thing

Well, you'd think so wouldn't you? but when I tried `this.stopListening.bind(this)` I'm pretty sure that it removed ALL the listeners! But then it was getting pretty late at that point so I may have made a simple mistake like forgetting to reload properly or doing so too quickly so the dev process hadn't finished... you know the sort of thing I'm sure ;-) So please feel free to give this a try for yourself and see what works and then I will re-test once you're happy.

**Comment by NayanKhedkar** [comment]
> ```js
> Adapt.on({
>     'app:dataReady': this.onDataReady.bind(this),
>     'app:languageChanged': function() {
>         this.stopListening();
>     }.bind(this)
> });
> ```

Worked fine.

File: js/adapt-contrib-pageLevelProgress.js [hunk 1]
```diff
@@ -9,7 +9,12 @@ define([
     var PageLevelProgress = Backbone.Controller.extend({
 
         initialize: function() {
-            this.listenTo(Adapt, 'app:dataReady', this.onDataReady);
+            Adapt.on({
+                'app:dataReady': this.onDataReady.bind(this),
+                'app:languageChanged': function() {
+                    this.stopListening();
```
Comment (oliverfoster) [inline js/adapt-contrib-pageLevelProgress.js]: Can you add a comment about this call removing on the events created by `setUpEventListeners`?
```js
                 'app:languageChanged': function() {
                    // Remove events created by setUpEventListeners
                    this.stopListening();
```

Comment (moloko) [reply js/adapt-contrib-pageLevelProgress.js]: might also be worth referencing [this issue](https://github.com/jashkenas/backbone/issues/4210) as to why the syntax is a little odd

Comment (NayanKhedkar) [reply js/adapt-contrib-pageLevelProgress.js]: 👍

**Pull request merged**